packages/clerk-js/src/ui/elements/AvatarUploader.tsx

```diff
@@ -8,7 +8,8 @@ import { FileDropArea } from './FileDropArea';
 export type AvatarUploaderProps = {
   title: LocalizationKey;
   avatarPreview: React.ReactElement;
-  onAvatarChange: (file: File) => Promise<unknown>;
+  onAvatarChange: (file: File | null) => Promise<unknown>;
+  hasImageUrl?: boolean;
```

Comment:
I think that the `hasImageUrl` prop doesn't really communicate its intent. What do you think about replacing this with an `onAvatarRemove?: () => Promise<unknown>` handler instead?

Reply:
@nikosdouvlis we want to check if avatar is the defualt gravatar one or a custom one. Maybe `hasDefaultImage`? I don't get how `onAvatarRemove` would work in this case